### ISSUE TITLE
[Feature] 

### ISSUE THREAD

**Issue by LonelySkullKid** [FrenzyRoleplay/server-issues#388]
Currently mechanic RP is on its arse, this is mainly because no one is coming to the shop for a car fix anymore so we can be and have been posted at the shop all day and apart from criminal items or us buying material we have nothing to do as a mechanic, this is due to bennys fixes being too strong so nerf what they can do & before a storm you could murder your car and have it at all 1s but as soon as that storm washes it away your car will come out of the garage again all 10s so again this gives mechanics no RP of any value. 
Suggested changes
1. Nerf bennys make the car barely fixed.
2. Stop the storm auto repairing cars if possible

Thank you for your time

**Comment by hhenry12** [comment]
What I can suggest to this, Benny only repairs the engine and body of the car. I believe this is how it is currently but I'm not sure. Codes or any of the other devs can speak on this. I remember us having a discussion in a community meeting but the way my memory is currently, I can't even remember yesterday.

**Comment by LonelySkullKid** [comment]
But it repairs the boxy and the engine 100% maybe make it less like 30/40% and most aren't bothering with mechanics fixing the other as the next storm will fix it for them

**Comment by Len77o** [comment]
Should also add that people should not be able to do vehicle upgrades at benny's (engine, brakes, turbo, etc...) Those should be done by a mechanic. So that option should either be removed or made crazy expensive.

**Comment by hhenry12** [comment]
> Should also add that people should not be able to do vehicle upgrades at benny's (engine, brakes, turbo, etc...) Those should be done by a mechanic. So that option should either be removed or made crazy expensive.

There is a difference between going to Benny and going to a mechanic in the city to get upgrades.

**Comment by iElainee** [comment]
I agree, I don’t know if currently Bennys only fixes Body and engine but it’s not expensive enough. And upgrades (turbo, engine, brakes..) also should be a mechanic Rp to do, just leave a couple customizations like window tint, respray to do yourself I guess, the Rp for mechanic is so poor and it’s sad. I’m not a mechanic but I wouldn’t like to be if it’s to do almost nothing.. It should be forced for civ to go see their mechanics to fix their cars and customize it. :)
